src/starterbot/BuildOrder.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <BWAPI.h>
+#include <vector>
+#include <string>
+
+// implements this: https://liquipedia.net/starcraft/Build_order
+
+class BuildOrder
+{
+
+public:
+
+	//// STATIC ////
+
+	// static map of build orders and their names
+	static std::map<std::string, BuildOrder> orders;
```

Comment:
This should not be static. When you call `BuildOrder::createBuildOrders()` in `StarterBot`, this will be initialized multiple times if the bot plays more than one game.

Instead, this should be a member of some other class, e.g. a `BuildingManager` class. Then, `StarterBot` can contain a `BuildingManager` member.

Reply:
I agree with this; please refer to the StarCraft Design Document for more information on structuring the BuildingManager class. 👍